test/e2e/README.md

```diff
@@ -7,10 +7,18 @@ End-to-end (e2e) testing is automated testing for real user scenarios.
 Prerequisites:
 - A running k8s cluster and kube config. We will need to pass kube config as arguments.
 - Have kubeconfig file ready.
-- Have Spark operator image ready.
+- Have a Kubernetes Operator for Spark image ready.
 
 e2e tests are written as Go test. All go test techniques apply (e.g. picking what to run, timeout length). Let's say I want to run all tests in "test/e2e/":
 
 ```bash
 $ go test -v ./test/e2e/ --kubeconfig "$HOME/.kube/config" --operator-image=gcr.io/spark-operator/spark-operator:v2.4.0-v1alpha1-latest
```

Comment:
I just realized that the image `gcr.io/spark-operator/spark-operator:v2.4.0-v1alpha1-latest` doesn't exist yet. Are you going to cut another release for the Operator with Spark 2.4?

Reply:
The image has already been built and pushed.

Reply:
```
$ docker pull gcr.io/spark-operator/spark-operator:v2.4.0-v1alpha1-latest
Error response from daemon: manifest for gcr.io/spark-operator/spark-operator:v2.4.0-v1alpha1-latest not found
```
I'm not able to pull it though. The 2.3.1 image is still there.

Reply:
Ah, looks like it was built but not pushed. Just pushed it.